.github/workflows/kubectl-plugin-release.yaml

```diff
@@ -1,21 +1,20 @@
 name: release-kubectl-plugin
 on:
   workflow_dispatch:
```

Comment:
Is it expected for `workflow_dispatch:` to be empty now?

Reply:
Yes. Now we should select "Tags" because `krew-release-bot` needs it.

![image](https://github.com/user-attachments/assets/2e716410-cd13-45e8-a9c3-d27a729aba39)